src/UnitTests/Analysis/CoalescerTests.cs

```diff
@@ -168,27 +168,43 @@ public void CoaSideEffectCalls()
         [Test]
         public void CoaCallUses()
         {
-            var m = new ProcedureBuilder("foo");
-            var r2 = m.Register(2);
-            var r3 = m.Register(3);
-            var r4 = m.Register(4);
+            var r2 = m.Reg32("r2");
+            var r3 = m.Reg32("r3");
+            var r4 = m.Reg32("r4");
             m.Assign(r4, m.Fn(r2));
-            m.Call(r3, 4);
-            m.Return();
-            RunFileTest(m, "Analysis/CoaCallUses.txt");
+            var uses = new Identifier[] { r2, r3, r4 };
+            var defines = new Identifier[] { };
+            m.Call(r3, 4, uses, defines);
+
+            RunCoalescer();
+
+            var expected =
+@"
+r4 = r2()
+call r3 (retsize: 4;)
+	uses: r2,r3,r4
+";
+            AssertProcedureCode(expected);
         }
 
         [Test]
         public void CoaCallCallee()
```

Comment:
I've been adding `[Category(Categories.UnitTests)]` to unit tests that don't require reading / writing from the file system. Now that you've gotten rid of the `*.exp` files for this test fixture, consider adding such `CategoryAttribute`

Reply:
Done in ecf88bc